bundle/manifests/customized-user-remediation.clusterserviceversion.yaml

```diff
@@ -70,7 +76,19 @@ spec:
         displayName: Template
         path: template
       version: v1alpha1
-  description: TBD
+  description: |
+    ### Introduction
+    Customized User Remediation (CUR) works with Node Health Check to remediate unhealthy nodes by running a user customized script on the unhealthy node.
+    This operator does not preform any fencing on the remediated node so users should be careful with the scripts that are being used.
+
+    ### Compatibility
+    CUR is one of the remediator operators by [Medik8s](https://www.medik8s.io/remediation/remediation/),
+    such as [Self Node Remediation](https://github.com/medik8s/self-node-remediation) and [Machine Deletion Remediation](https://github.com/medik8s/machine-deletion-remediation),
```

Comment:
Any reason to not mention FAR as well?

Reply:
>Any reason to not mention FAR as well?

Not something in particular, I assume it's because it's copy pasted from FAR :-)

Reply:
>We might even want to update this similar sentence in NHC CSV for more engagements of CUR by NHC users.

Probably it's something we want to avoid at least until CUR will have an Official Red Hat release.

Reply:
Even for the upstream CSV? Eventually, we plan CUR to be in OperatorHub first. So there is a (small) value for that.

Reply:
Since CUR at the moment is more of an "experimental" operator I think I prefer to keep NHC "clean".
Also, ATM in case we want NHC to mention CUR only downstream it will also require some changes to make sure only NHC upstream CSV is affected (so either changes to render_template, or a make job).
I don't think it worth it now, probably will be more relevant in case CUR will grow in maturity.

Reply:
SGTM